src/SFA.DAS.DownloadService.UnitTests/Web.Controllers/AparControllerTests.cs

```diff
@@ -33,23 +33,77 @@ public async Task Csv_NoRecordsReturnedFromApi_RedirectToServiceUnavailable()
             _mockClient.Setup(x => x.GetAparSummary()).ReturnsAsync(new List<AparEntry>());
             var result = await _controller.DownloadCsv();
             var redirectResult = result as RedirectToActionResult;
-            Assert.AreEqual("ServiceUnavailable",redirectResult.ActionName);
-            _mockClient.Verify(x=> x.GetAparSummary(),Times.Once);
+            Assert.AreEqual("ServiceUnavailable", redirectResult.ActionName);
+            _mockClient.Verify(x => x.GetAparSummary(), Times.Once);
```

Comment:
If it makes no sense for line 37 to run if line 36 fails, this is fine. Otherwise, this should be inside an `Assert.Multiple`